### PR TITLE
Add Face ID usage description to ValetTouchIDTest app

### PR DESCRIPTION
**Pull request by NickEntin**

### CODE REVIEW

**Comment by dfed** [comment]
LGTM! May be nifty to make the string value more instructional, but string search will certainly work.

**Comment by dfed** [comment]
FYI @EricMuller22 we'll want to adopt a similar change on #80 since we're rewriting a lot of this there.

File: README.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ VALSinglePromptSecureEnclaveValet *mySecureEnclaveValet = [[VALSinglePromptSecur
 
 This instance also stores and retrieves data in the Secure Enclave, but does not require the user to confirm their presence each time data is retrieved. Instead, the user will be prompted to confirm their presence only on the first data retrieval. A `VALSinglePromptSecureEnclaveValet` instance can be forced to prompt the user on the next data retrieval by calling the instance method `requirePromptOnNextAccess`.
 
+**In order for your app to officially support Face ID, you must set a value for the Privacy - Face ID Usage Description (NSFaceIDUsageDescription) key in your app's Info.plist.**
```
Comment (dfed) [inline README.md]: I'd say: "In order for your customers not to receive a prompt that your app does not yet support Face ID, you must ..."

Also, curly quotes 😉

**Comment by EricMuller22** [comment]
Gorm

**Pull request merged**